setup.py

```diff
@@ -23,5 +23,6 @@
         'pyarrow>=0.8.0',
         'pyspark',
         'scipy',
+        'setuptools-scm',
```

Comment:
I think this should go in:
`    setup_requires=['pytest-runner', 'setuptools_scm'],`